test-files/test_gap_gc_request_signal_rooting.ts

```
@@ -0,0 +1,38 @@
+  return keep.length;
+}
+
+const req = new Request("https://example.test/x", { method: "POST", body: "b" });
+churn();
+// Pre-fix: req.signal returns the registry's stale bits; property reads on
+// it dereference the dead/moved object.
+console.log("signal:", typeof req.signal, req.signal.aborted);
+const clone = req.clone();
+churn();
+console.log("clone signal:", typeof clone.signal, clone.signal.aborted);
+console.log("same object:", req.signal === req.signal);
```

Biome (2.5.6)

[error] This comparison uses the same expression on both sides.

(lint/suspicious/noSelfCompare)